easybuild/easyblocks/i/icc.py

```diff
@@ -96,7 +103,7 @@ def sanity_check_step(self):
 
         custom_paths = {
             'files': ["%s/%s" % (binprefix, x) for x in binfiles] +
-                     ["%s%s" % (libprefix, x) for x in ["iomp5.a", "iomp5.so"]],
+                     ["%s/lib%s" % (libprefix, x) for x in ["iomp5.a", "iomp5.so"]],
```

Comment:
this should use `os.path.join` (mostly because of code style):

```python
[os.path.join(binprefix, x) for x in binfiles] +
[os.path.join(libprefix, 'lib%s' % x) for x in ['iomp5.a', 'iomp5.so']],
```